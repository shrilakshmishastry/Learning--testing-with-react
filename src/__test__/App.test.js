import React from 'react';
import App from './../App.js';
import {render,cleanup} from '@testing-library/react';


it("should take snapshot of app",()=>{
  const {asFragment} = render(<App/>)
  expect(asFragment(<App/>)).toMatchSnapshot()
})
