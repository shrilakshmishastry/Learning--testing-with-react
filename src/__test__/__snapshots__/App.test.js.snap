// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take snapshot of app 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <h1>
      Testing Updated
    </h1>
  </div>
</DocumentFragment>
`;
